server/services/personService.test.ts

```diff
@@ -0,0 +1,75 @@
+import createError from 'http-errors'
+
+import PersonService from './personService'
+import { PrisonerClient } from '../data'
+import { prisonerFactory } from '../testutils/factories'
+import personUtils from '../utils/personUtils'
+
+jest.mock('../data/prisonerClient')
+
+describe('PersonService', () => {
+  const prisonerClient = new PrisonerClient('token') as jest.Mocked<PrisonerClient>
+  const prisonerClientBuilder = jest.fn()
+
+  const service = new PersonService(prisonerClientBuilder)
+
+  const token = 'token'
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+    prisonerClientBuilder.mockReturnValue(prisonerClient)
+  })
+
+  describe('getPerson', () => {
+    describe('when the prisoner client finds the corresponding prisoner', () => {
+      it('converts the returned object to a Person', async () => {
+        const prisoner = prisonerFactory.build()
+
+        const person = personUtils.personFromPrisoner(prisoner)
```

Comment:
This differs a little from the organisation equivalent, i.e. it's using person utils to do the same thing that the service does - I wonder if this is replicating logic rather than testing the outcome?

Reply:
I've realised we should really be using mocks here, and in our controller tests to prevent us re-testing these types of methods everywhere. We should trust the behaviour of `personUtils.personFromPrisoner` works, and is tested elsewhere. I'll tidy up my branch to do this some more, and add some cards for refactoring earlier test.

Reply:
Sounds good - curious to see what the revised approach looks like!